themes/default/content/blog/cloud-systems-part-one/index.md

```diff
@@ -0,0 +1,296 @@
+    <meta charset="utf-8">
+    <title>Hello world!</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="normalize.css">
+</head>
+<body>
+    <header>
+        <div class="logo"><i class="fas fa-cat"></i></div>
+        <ul class="social">
+                <li><a href="http://github.com/katcosgrove" target="_blank"><i class="fab fa-github-alt"></i></a></li>
+                <li><a href="http://twitter.com/Dixie3Flatline" target="_blank"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="http://linkedin.com/in/katcosgrove" target="_blank"><i class="fab fa-linkedin-in"></i></a></li>
+            </ul>
+    </header>
+<div class="banner">
+    <h1>Your Name</h1>
+    <h3>Job Title at Company</h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam blandit tristique arcu, sed commodo ligula. Ut ullamcorper velit non luctus mollis. Sed placerat quam sed tellus aliquam, ac eleifend quam lacinia. Praesent bibendum velit at metus auctor, vitae feugiat quam luctus. Morbi eu imperdiet metus, et tincidunt tellus. Aliquam non ullamcorper justo. Cras ornare nulla vel pellentesque vulputate. Mauris in elit neque. Aliquam tempor aliquam libero, elementum luctus nisl imperdiet sit amet. Praesent urna ante, scelerisque non tellus mollis, laoreet sodales felis.</p>
+</div>
+</body>
+<script src="https://kit.fontawesome.com/b4747495ea.js" crossorigin="anonymous"></script>
+</html>
+```
+
+`style.css`
+
+```css
+@import url('https://fonts.googleapis.com/css?family=News+Cycle|Teko&display=swap');
+
+ul {
+    list-style-type: none;
+}
+
+ul li {
+    display: inline-block;
+}
+
+a {
+    color: white;
+    -webkit-transition: color .5s ease-out;
+    transition: color .5s ease-out;
+    text-decoration: none;
+}
+
+a:hover, a:active {
+    color: rgb(255, 157, 112);
+}
+
+header {
+    background-color: rgba(0, 0, 0, .8);
+    height: 80px;
+    position: absolute;
+    top: 0;
+    width: 100%;
+}
+
+header li {
+    color: white;
+}
+
+.active a {
+    color: rgb(255, 157, 112);
+}
+
+.social {
+    position: absolute;
+    right: 50px;
+    top: -5px;
+    font-size: 30px;
+}
+
+.social li {
+    margin: 0 5px 0 5px;
+}
+
+.logo {
+    position: absolute;
+    left: 10px;
+    margin-left: 20px;
+    font-size: 60px;
+    color: white;
+}
+
+body {
+    background: url('./background.jpg');
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.banner {
+    width: 60vw;
+    font-family: Teko;
+    font-size: 2vw;
+    text-align: center;
+    margin-top: 15vw;
+    margin-left: 20vw;
+}
+
+.banner p {
+    font-family: News Cycle;
+}
+```
+
+Your `normalize.css` can be copied from [this respository](github.com/necolas/normalize.css), and the expected `background.jpg` can be downloaded from [this repository](https://github.com/katcosgrove/cloud-systems-101/blob/main/part-one/website/background.jpg).
+
+To see what this looks like before we deploy it, install live-server:
```

Comment:
```suggestion
To see what this looks like before we deploy it, install `live-server`:
```

Reply:
Also, I'm thinking it might be a good idea to note that this is part of the optional install of Node (i.e., if they skipped that dependency, they can't run this).